SweatEquity/IBM/README.md

```diff
@@ -1,8 +1,52 @@
 # IBM Sweat Equity
 
-put stuff here about stixshifter and kestrel
+## Projects Description
 
-and Open Horizon wrt One Planet Education Network request for SBOMs
+### stix-shifter (OCA)
+STIX-shifter is an open-source python library that queries security products and data stores using STIX patterning and presents the results as STIX cyber observable objects.  The project contains several connectors such as Splunk, QRadar, and Crowdstrike Falcon. 
```

Comment:
Can we get a link in here?
`[STIX-shifter](https://github.com/opencybersecurityalliance/stix-shifter)`

Reply:
I merged it in so people could see it. You could make another PR with the link if you want